src/lesson_4/script.js

```diff
@@ -0,0 +1,97 @@
+let sum = 0.1 + 0.2;
+console.log( sum.toFixed(2) );
+
+const userLogin = prompt("укажите свой логин");
+console.log("userLogin =", userLogin );
+
+const userPassword = prompt("введіть свій пароль?");
+console.log("userPassword =", userPassword );
+
+if(userLogin != 'admin' && userPassword != '12pass33210') {
```

Comment:
```suggestion
if(userLogin !== 'admin' && userPassword !== '12pass33210') {
```
Please, use strict equality operator